src/AWS/Lambda/Runtime/Value.hs

```diff
@@ -331,7 +332,7 @@ ioRuntime = ioRuntimeWithContext . withoutContext
 -- @
 fallibleRuntimeWithContext :: ToJSON result =>
   (LambdaContext -> Value -> Either String result) -> IO ()
-fallibleRuntimeWithContext = mRuntimeWithContext . fmap (fmap pure)
+fallibleRuntimeWithContext fn = mRuntimeWithContext (\lc -> either error pure . fn lc)
```

Comment:
Makes sense, because we don't want to use the instance for `Either`s. But if we're using `mRuntimeWithContext`, we're in a `MonadCatch`; maybe we could throw a better exception?

Reply:
The entire package is using `error`, I don't want my changes to leave the package in an inconsistent state (or change all occurance of `error`)

Reply:
Fair.